fdbclient/ServerKnobs.cpp

```diff
@@ -467,6 +467,8 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( PROXY_REJECT_BATCH_QUEUED_TOO_LONG,                    true );
 
 	bool buggfyUseResolverPrivateMutations = randomize && BUGGIFY && !ENABLE_VERSION_VECTOR_TLOG_UNICAST;
+	if (ENABLE_VERSION_VECTOR_TLOG_UNICAST == true) buggfyUseResolverPrivateMutations = true;
+
 	init( PROXY_USE_RESOLVER_PRIVATE_MUTATIONS,                 false ); if( buggfyUseResolverPrivateMutations ) PROXY_USE_RESOLVER_PRIVATE_MUTATIONS = deterministicRandom()->coinflip();
```

Comment:
Should this always be true when unicast (or VV) is enabled?

Reply:
Yes.

Reply:
this is a bug in my PR, I'll fix it.